tidy3d/components/medium.py

```diff
@@ -1530,6 +1555,51 @@ def eps_diagonal(self, frequency: float) -> Tuple[complex, complex, complex]:
         eps_spatial_array = (_get_numpy_array(eps_comp).ravel() for eps_comp in eps_spatial)
         return tuple(eps_comp[np.argmax(np.abs(eps_comp))] for eps_comp in eps_spatial_array)
 
+    def _eps_bounds(
+        self, frequency: float = None, eps_component: Optional[PermittivityComponent] = None
+    ):
+        """Returns permittivity bounds for setting the color bounds when plotting.
+
+        Parameters
+        ----------
+        frequency : float = None
+            Frequency to evaluate the relative permittivity of all mediums.
+            If not specified, evaluates at infinite frequency.
+        eps_component : Optional[PermittivityComponent] = None
+            Component of the permittivity tensor to plot for anisotropic materials,
+            e.g. ``xx``, ``yy``, ``zz``, ``xy``, ``yz``, ...
+            Defaults to ``None``, which returns the average of the diagonal values.
+
+        Returns
+        -------
+        Tuple[float, float]
+            The min and max values of the permittivity for the selected component and evaluated at ``frequency``.
+        """
+        eps_dataarray = self.eps_dataarray_freq(frequency)
+
+        comps = ["xx", "yy", "zz"]
+        if eps_component in comps:
+            # Return the bounds of a specific component
+            eps = _get_numpy_array(np.real(eps_dataarray[comps.index(eps_component)])).ravel()
```

Comment:
think it might be worth defining a function in the scope of this method.

```py
def get_eps_values(x: np.ndarray) -> np.ndarray:
    """Grab the real part of the values in array"""
    return _get_numpy_array(np.real(x)).ravel()
```

because this is repeated a few times.